scrapscript.py

```diff
@@ -575,6 +584,9 @@ def deserialize(msg: Dict[str, object]) -> "Bytes":
         assert isinstance(msg["value"], bytes)
         return Bytes(msg["value"])
 
+    def __str__(self) -> str:
```

Comment:
We should stringify the way it appears in the source (as hex, etc)

Reply:
Makes sense. By the time we have evaluated the expression we've lost the base information, so we can't convert back to base 16, 32, etc byte strings, but we can at least match the base64 format so that if the user copies the output they can input it to get the same value.